libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialog.cs

```diff
@@ -406,6 +464,74 @@ protected async Task<bool> ProcessEventAsync(SequenceContext sequenceContext, Di
             return handled;
         }
 
+        protected async Task<bool> ProcessFormAsync(SequenceContext sequenceContext, CancellationToken cancellationToken)
```

Comment:
>ProcessFormAsync [](start = 35, length = 16)

should this be virtual?  protected method implies contract to child classes.

Reply:
Made it private and opened work item to review this.

---
In reply to: [360163692](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360163692) [](ancestors = 360163692)